app/test_main.py

```diff
@@ -1 +1,29 @@
-# write your code here
+import pytest
+
+from unittest.mock import patch
+from app.main import can_access_google_page
```

Comment:
Order and group your imports according [to PEP-8](https://peps.python.org/pep-0008/#imports), `unittest` is a standard module, `pytest` is third party and `app` is related to a local application